docs/hooks/patient-view.md

```diff
@@ -12,14 +12,18 @@ The user has just opened a patient's record.
 
 ## Context
 
-The patient whose record was opened, including their encounter, if applicable.
+The patient whose record was opened, including their encounter, if applicable. 
+
+The FHIR ids referenced in `context` are expected to be available via the FHIR Server (if `fhirServer` is populated in the request) or via prefetch, if supported. However, the specification does not explicitly require this behavior and implementations should be aware that integrations may not fully support this expectation.
 
 Field | Optionality | Prefetch Token | Type | Description
 ----- | -------- | ---- | ---- | ----
 `userId` | REQUIRED | Yes | *string* | The id of the current user.<br />For example, if the user represents a FHIR resource on the given FHIR server, the resource type would be one of [Practitioner](https://www.hl7.org/fhir/practitioner.html), [Patient](https://www.hl7.org/fhir/patient.html), or [RelatedPerson](https://www.hl7.org/fhir/relatedperson.html).<br />If the user was a Practitioner, this value would be `Practitioner/123`
 `patientId` | REQUIRED | Yes | *string* | The FHIR `Patient.id` of the current patient in context
 `encounterId` | OPTIONAL | Yes | *string* | The FHIR `Encounter.id` of the current encounter in context
 
+For information on patient safety issues see [Security & Safety](../../specification/1.0/#security-and-safety).
```

Comment:
Suggest:  
The patient-view hook only allows advice that is based on information provided using prefetch and/or the FHIR-server. As is stated in  [Security & Safety], implementers should be aware that using the patient-view hook might result in a decision based on outdated patient data, resulting in a safety risk to the patient. E.g. because the data the advice has been based on has been altered in the current user-session.

Reply:
Does this addition provide any information that is not already available in the base spec?

Reply:
Dennis - that's a good question. Bas, how much of the base spec should we be repeating here?

Reply:
Adding comments about outdated/stale clinical data and a link to the Security and Safety page is redundant since the 1.0 specification [already calls this out](https://cds-hooks.hl7.org/1.0/#security-and-safety)

> Security and safety risks associated with the CDS Hooks API include:
> ...
> 6. The risk that a CDS Service could return a decision based on outdated patient data, resulting in a safety risk to the patient.

@bvdh, are you agreeable to removing this language from the hook since it's already covered by the published base spec?